### ISSUE TITLE
Strip the inner HTML tags

### ISSUE THREAD

**Issue by vickyRathee** [AngleSharp/AngleSharp#470]
I was in a need to remove the inner html tags such as br and span from paragraph. I can use the TextContent but the problem is the HTML is badly formatted and these tags represent the space and line breaks. So using TextContent results in two words merged.

I'm really close but stuck with how to use the the ReplaceChild to replace these inner tags with space or other delimiters ?
```
           var source = @"<p>Strip the<br/>inner break html tag<p>";

            var parser = new HtmlParser();
            var document = parser.Parse(source);
            var Selector = document.QuerySelector("p");

            var nodes = Selector.Children;
            foreach (var node in nodes)
            {
                Selector.ReplaceChild(<What should be here>, node);
            }
            
            Console.Write(Selector.TextContent);
```

**Comment by vickyRathee** [comment]
**Example 2**
```
 var source = @"<p>Strip the<br/>inner tags<span class="line-break">some value</span><p>";
```

**Comment by FlorianRappl** [comment]
`replaceChild` is simple. The API documentation for this can be found, e.g., [here](https://developer.mozilla.org/de/docs/Web/API/Node/replaceChild).

That being written - I guess this is not what you need / want. You want to present your HTML as text nicer than by just using `TextContent`, which also contains the formatting newlines and tabs.

AngleSharp does not contain such a function or helper - it would be highly opinionated. Maybe such a helper will come in one form or the other, but right now no such plans exist.

So long story short: Get the correct DOM from AngleSharp and walk over the nodes to identify and remove nodes from a blacklist (or keep those from a whitelist).
